.vscode/launch.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Just a minor thing - these kind of files are used to control your IDE, in this case vscode. We'd normally add these files to your .gitignore. In a real project, it is probably the case that developers might want different settings in their IDE.

You'd do this by adding `.vscode/*` to your `.gitignore` file. 

Again this isn't something you need to do now, it's just for information